securedrop/tests/test_store.py

```diff
@@ -261,6 +261,29 @@ def test_add_checksum_for_file(config, app_storage, db_model):
         assert db_obj.checksum == "sha256:" + expected_hash
 
 
+def _wait_for_redis_worker(job: Job, timeout: int = 60) -> None:
```

Comment:
I moved this function here and made it "private" since it's only used in this file.